conductor_api/src/conductor/debug.rs

```diff
@@ -0,0 +1,34 @@
+use conductor::Conductor;
+use holochain_core::state_dump::StateDump;
+use holochain_core_types::error::HolochainError;
+use holochain_persistence_api::cas::content::Address;
+
+pub trait ConductorDebug {
+    fn running_instances(&self) -> Result<Vec<String>, HolochainError>;
+    fn state_dump_for_instance(&self, instance_id: &String) -> Result<StateDump, HolochainError>;
+    fn get_type_and_content_from_cas(
```

Comment:
Could use `HcResult` instead of `Result` here and several other places, but not terribly important.